benchmarks/MongoDB.Driver.Benchmarks/BenchmarkHelper.cs

```diff
@@ -79,13 +78,13 @@ public static class MongoConfiguration
             public const string PerfTestDatabaseName = "perftest";
             public const string PerfTestCollectionName = "corpus";
 
-            public static DisposableMongoClient CreateDisposableClient()
+            public static IMongoClient CreatClient()
```

Comment:
Just happened to notice this typo while perusing the repo. Not sure how much it matters! Will require changes in all files, of course

```suggestion
            public static IMongoClient CreateClient()
```

Reply:
Thanks, done!